index.html

```diff
@@ -57,8 +57,15 @@ <h3 class="main-subheading">
         assets at your fingertips.
```

Comment:
You need to create wrap for "hero" section (first screen of the site) for group elements